### ISSUE TITLE
[Feature Request] Change File Log location directory

### ISSUE THREAD

**Issue by qurious-pixel** [Ryujinx/Ryujinx#1526]
#### Overview:
Move logging out of Publish folder into config folder

#### Smaller Details: 
Placing logging in a known writeable area.

#### Nature of Request:  
- Change
Ryujinx/Ui/SettingsWindow.cs
Ryujinx.Common/Logging/Targets/FileLogTarget.cs

### Why would this feature be useful?
Sandboxed apps such as AppImages and FlatPaks cannot write to the read-only section.
This crashes the emulator.

**Comment by macabeus** [comment]
Related PR: https://github.com/Ryujinx/Ryujinx/pull/1530

**Comment by qurious-pixel** [comment]
closing this since it can be selected `return AppDataManager.BaseDirPath;` by setting ChannelOwner.

      private const string FlatHubChannelOwner = "flathub";

**Comment by TSRBerry** [comment]
Yes, but I still think we shouldn't log to the "installation" directory in any case. So I'd like to keep your feature request around for that if that's fine for you!
